news/2019-02-22-acknowledging-community-contributors.md

```diff
@@ -0,0 +1,201 @@
+---
+layout: post
+title: "Acknowledging Community Contributors"
+date: 2019-02-22 00:00:00 +0000
+---
+
+At its very heart, osu! is almost entirely driven by its community. And at the heart of our community, the efforts of an outstanding few make themselves known. Today, we take a moment to reflect on some of these people, and award them the recognition they deserve.
+
+![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/banner.jpg)
+
+Perhaps the most prestigious of all profile badges awarded, the [community contributor badge](https://assets.ppy.sh/profile-badges/contributor.png) is expressly granted to those of the osu! community who have gone utterly above and beyond the norm in their enduring contribution to the well-being of the game and its community.
```

Comment:
Is the badge supposed to be this one? https://assets.ppy.sh/profile-badges/contributor.jpg

Reply:
Good catch!